website/content/docs/install/cloud-auto-join.mdx

```diff
@@ -138,12 +142,20 @@ $ consul agent -retry-join "provider=aws tag_key=... tag_value=..."
 - ECS task role metadata (container-specific).
 - EC2 instance role metadata.
 
-The only required IAM permission is `ec2:DescribeInstances`, and it is
+To discover EC2 consul-servers, the only required IAM permission is `ec2:DescribeInstances`, and it is
```

Comment:
```suggestion
The only IAM permission required for discovering EC2 consul-servers is `ec2:DescribeInstances`. It is
```
Ideally, we'd start the second sentence with "We recommend . . . " so that it is in active voice per our style guide, but I don't think I can make those suggestions here.